### DISCUSSION TITLE
Updating sparkle using carthage

### DISCUSSION THREAD

**Discussion by prasann-1433** [sparkle-project/Sparkle#2372] [General]
As per the official documentation, i am trying to update sparkle using cartfile, everything works fine but even though i give a version i am interested in, it always download the 2.4.1. May be carthage dependencies is doing some trick, but is there a way i can ask this specific version only to download not the latest 2.4.1

Was tring using this, but cartfile resolved always show 2.4.1
binary "https://sparkle-project.org/Carthage/Sparkle.json" ~> 2.4.0.

Why i am interested in 2.4.0 --> no specific reason but want to make sure i don't pick up the latest updated stuff at the last day, so just a way to delay on our side.

**Comment by zorgiepoo** [comment]
My understanding of Carthage: `~> 2.4.0` means that `2.4.0` is compatible with `2.4.1` so it will prefer `2.4.1` (since `2.4.1` is a patch-fix update). That works as expected. However if `2.5.0` is released, then `~> 2.4.0` should stay at `2.4.1`. If you want specifically and only `2.4.0` you would use `== 2.4.0`. There is also `>=` if you only care about "at least."

**Comment by prasann-1433** [comment]
hmm ok, not aware about this. Thanks for this tip.
